server/controllers/stock/core.js

```diff
@@ -608,7 +615,15 @@ async function getDailyStockConsumption(params) {
     filters.setGroup('GROUP BY DATE(m.date)');
   }
 
-  filters.setOrder('ORDER BY m.date ');
+  if (params.group_by_flux) {
+    filters.setGroup('GROUP BY m.flux_id');
```

Comment:
I think this needs to group by `m.flux_id` and also `m.document_uuid`.  Either that, or you need to only `COUNT(DISTINCT document_uuid)` somewhere.  Although due to https://github.com/IMA-WorldHealth/bhima/issues/4083, counting unique documents may not work in this case.

To illustrate, here is the stock movement registry for IMCK:
![image](https://user-images.githubusercontent.com/896472/109868001-6ef5a480-7c67-11eb-8755-99d72e928316.png)

There are only 14 movements from purchase orders.

And here is the graphical report:
![image](https://user-images.githubusercontent.com/896472/109868124-8f256380-7c67-11eb-9239-dd1c193750c1.png)

This thinks there is 56 movements.  My assumption is the issue is that we are counting the individual lines instead of the distinct documents.